src/components/Envelope.vue

```diff
@@ -1025,4 +1029,25 @@ export default {
 :deep(.list-item__extra) {
 	margin-top: 9px;
 }
+:deep(.line-two__subtitle) {
+	display: flex;
+	width: 450px;
+}
+:deep(.list-item-content__main) {
+	flex-direction: row;
+	display: flex;
+	flex-wrap: nowrap;
+	justify-content: flex-start
+}
+:deep(.line-one__title) {
+	flex-direction: row;
+	display: flex;
+	width: 200px;
+}
+:deep(.line-two) {
+	flex-grow: 1;
+}
+:deep(.line-one__details) {
+	float: right;
+}
```

Comment:
this is the part that should be upstreamed. the app list item needs at least two modes: one for the holizontal split (what we have now) or narrow screens and one for full width